Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,30 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            for (int i = 0; i < 2; i++)
+            {
+                Console.WriteLine("Enter person " + (i+1) + " first name:");
+                string fName = Console.ReadLine();
+
+                Console.WriteLine("Enter " + fName + "'s last name:");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define a constant instead of using this literal 'Enter ' 4 times.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4884999)